examples/inference/HF_inference.py

```diff
@@ -6,6 +6,7 @@
 import time
 
 import torch
+import pippy
 import pippy.fx
 from pippy import run_pippy
 from pippy.IR import MultiUseParameterConfig, Pipe
```

Comment:
This import is not needed anymore, I think.

Reply:
Neither do we need
`from pippy.PipelineDriver import PipelineDriverFillDrain ...`